sql/src/main/java/io/crate/planner/node/dql/RoutedCollectPhase.java

```diff
@@ -76,7 +81,8 @@ public RoutedCollectPhase(UUID jobId,
                               List<Symbol> toCollect,
                               List<Projection> projections,
                               WhereClause whereClause,
-                              DistributionInfo distributionInfo) {
+                              DistributionInfo distributionInfo,
+                              @Nullable User user) {
```

Comment:
please add an assertion to the stream methods that the user must be null since it's not streamable.